models/user.js

```diff
@@ -9,19 +9,15 @@ const userSchema = new Schema({
   googleId: { type: String, unique: true, required: true },
   photos: [],
   userSocialMedia: {
-    instagram: {
```

Comment:
don't we still need a location to store off the new contacts links to instagram/facebook/linkedin? so the buttons they click send them to the proper account location?